docs/content/design-keuzes.md

```diff
@@ -0,0 +1,37 @@
+# Designkeuzes
+12345678901234567890123456789012345678901234567890123456789012345678901234567890
+## API endpoints gebruiken UUID4 als ID-parameter voor het opvragen van een enkel object
+
+De ID-parameter, hieronder aangeduid met `{id}` wordt gebruikt om via de URL 
```

Comment:
@joeribekker die wil ik wel gaan hernoemen naar `{uuid}` - meteen aanpassen hier, of wachten tot de referentiecomponenten het ook daadwerkelijk gebruiken?

Reply:
Eens.